File: Stylpic/Stylpic/SPPhotoPair.m
//
//  SPPhotoPair.m
//  Stylpic
//
//  Created by Joshua Bell on 3/22/15.
//  Copyright (c) 2015 Neil Bhargava. All rights reserved.
//

#import "SPPhotoPair.h"
#import <Parse/PFObject+Subclass.h>

@implementation SPPhotoPair

@dynamic photoOne, photoTwo, caption, user, isArchiveReady;

+ (void)load {
    [self registerSubclass];
}

+ (NSString *)parseClassName {
    return @"PhotoPair";
}


@end

File: Stylpic/Stylpic/SPPhotoPair.swift
//
//  SPPhoto.swift
//  Stylpic
//
//  Created by Neil Bhargava on 3/1/15.
//  Copyright (c) 2015 Neil Bhargava. All rights reserved.
//

import UIKit

class SPPhotoPair: PFObject, PFSubclassing {
    @NSManaged var imageOne : PFFile?
    @NSManaged var imageTwo : PFFile?
    @NSManaged var thumbnailOne : PFFile?
    @NSManaged var thumbnailTwo : PFFile?
    @NSManaged var caption : String?
    @NSManaged var user : SPUser?
    
    override init() {
        super.init()
    }
//    func setup( imageOne: PFFile, thumbnailImageOne: PFFile, imageTwo: PFFile, thumbnailImageTwo: PFFile, postCaption: String, postingUser: SPUser ) {
//        self.imageOne = imageOne
//        self.imageTwo = imageTwo
//        self.thumbnailOne = thumbnailImageOne
//        self.thumbnailTwo = thumbnailImageTwo
//        self.caption = postCaption
//        self.user = postingUser
//    }
    
    override class func initialize() {
        var onceToken : dispatch_once_t = 0;
        dispatch_once(&onceToken) {
            self.registerSubclass()
        }
    }
    
    static func parseClassName() -> String! {
        return "Photos"
    }
    
}
